### DISCUSSION TITLE
Hero 11 Wake on Voice

### DISCUSSION THREAD

**Discussion by fch4um0n7** [gopro/labs#216] [Feature Requests]
Hey there,

On my Hero 7 I'm very fond of voice commands with Wake on Voice enabled which I find very convenient when surfing to start/stop recording without the need to reach out for the camera and hit the quick capture button.
On my brand new Hero 11 I'm quite disappointed that the Wake on Voice is not there anymore.

I heard some folks complained about quicker battery drainage but since it can be disabled when not needed I don't really see the problem.

So out of curiosity:
- What's the reason for dropping this option?
- Do you plan on bringing it back ever?

Thanks.

(Sorry if this isn't the right place to inquire about it. If so please redirect me accordingly)

**Comment by dnewman-gpsw** [comment]
Not the right place, this is for Labs firmware discussions.  The HERO11 forum on GoPro.com is here: https://community.gopro.com/s/topic/0TO3b000000K1RTGA0/hero11-black

**Reply by fch4um0n7** [reply]
Awesome. Thank you.

**Comment by fch4um0n7** [comment]
Forgot to close it...
